hedera-node/hedera-app/src/main/java/com/hedera/node/app/blocks/impl/BlockStreamBuilder.java

```diff
@@ -1032,8 +1032,6 @@ public void nullOutSideEffectFields() {
         transactionResultBuilder.scheduleRef((ScheduleID) null);
         transactionResultBuilder.automaticTokenAssociations(emptyList());
         transactionResultBuilder.congestionPricingMultiplier(0);
-        transactionResultBuilder.status(ResponseCodeEnum.OK);
-        transactionResultBuilder.parentConsensusTimestamp(Timestamp.DEFAULT);
 
```

Comment:
I'm ok with this, as long as we're sure to set these properties in all relevant cases